versioning/BRM/brmtypes.h

```diff
@@ -163,6 +163,36 @@ struct CPInfo
         int64_t min_;
     };
     bool isBinaryColumn;
+    /** @brief Set CPInfo into "invalid range" value.  It relies on the isBinaryColumn to be already set. */
+    void toInvalid(bool isColTypeUnsigned) // XXX Maybe we better switch on the column type instead???
+    {
+        if (!isBinaryColumn)
+        {
+            if (isColTypeUnsigned)
+            {
+                max = numeric_limits<uint64_t>::min();
+                min = numeric_limits<uint64_t>::max();
+            }
+            else
+            {
+                max = numeric_limits<int64_t>::min();
+                min = numeric_limits<int64_t>::max();
+            }
+	}
+	else
+	{
+            if (isColTypeUnsigned)
+            {
+                bigMax = 0;
+                bigMin = -1;
+            }
+            else
+            {
+                utils::int128Min(bigMax);
```

Comment:
Better replace this with datatypes::numeric_limits<int128_t>max() taken from ./datatypes/mcs_numeric_limits.h

Reply:
It was moved to we_brm.h (ExtCPInfo::toInvalid()) and now uses mcs_datatypes.h-defined ranges.